test-d/index.test-d.ts

```diff
@@ -0,0 +1,71 @@
+import { expectType, expectAssignable, expectError } from 'tsd';
+
+import { KEYS, getKeys, unionWith, VisitorKeys } from "../";
+
+type VisitorKeysWritable = { [type: string]: ReadonlyArray<string> };
```

Comment:
Why do we need this in the tests?

Reply:
Was just for convenience--like defining a `const` in a Mocha test. There's no `export` on this `type`, so it seems to me to fit with the testing approach here.

Reply:
No problem with that, but what are we testing with `VisitorKeysWritable`?

Reply:
I see. Yes, the tests had indeed only been relevant before because we were actually exporting the type. Now removed.